content/community/news-events.md

```diff
@@ -16,7 +16,7 @@ abstract: "Get the latest news and info about upcoming events"
 
 **2020**
 
-- Dec. 1, 2020: Quire [newsletter](https://newsletters.getty.edu/t/t-DFDC53BEAACB81BA2540EF23F30FEDED) and [community forum](https://github.com/thegetty/quire/discussions) are launched
+- Dec. 1, 2020: Quire [newsletter](https://mailchi.mp/d030942ba347/quire-a-year-in-review?e=5c4361e9ac) and [community forum](https://github.com/thegetty/quire/discussions) are launched
```

Comment:
Not sure how this mailchimp one got in here, but I assume you don't want it

Reply:
Rather than have two links to the sign-up I thought it might be nice to link to the inaugural newsletter that was launched on Dec. 1.

Reply:
Oh! I see. I'm just seeing that the Subscribe buttons and other stuff are still active. I guess on the off chance someone tried to subscribe there you'd get a notification?

Reply:
Ah, ok. good point. That is a bit confusing for people. I'll switch it to the new sign-up link.

What do you think about making "sign up to receive updates" as a button rather than italicized? 

I'm also checking with Kathie to see if we can get a link to an archive of sent emails instead. This would eliminate the need for "check out our most recent newsletter," which might be a pain to keep updated.

Reply:
👍 

Sure could be a button. Will approve and you can adjust and merge in whenever you're ready